security/capabilities/capabilities.go

```diff
@@ -11,7 +9,7 @@ const allCapabilityTypes = capability.CAPS | capability.BOUNDS
 // DropBoundingSet drops the capability bounding set to those specified in the
 // container configuration.
 func DropBoundingSet(capabilities []string) error {
-	c, err := capability.NewPid(os.Getpid())
+	c, err := capability.NewPid(0)
```

Comment:
Change capabilities of the calling process. os.Getpid() and 0 are the same except the case, when you try to access /proc from another pidns